.github/workflows/ci.yaml

```diff
@@ -14,9 +14,9 @@ jobs:
           go-version: "1.21.x"
       - uses: actions/checkout@v4
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
+        uses: golangci/golangci-lint-action@v4
         with:
-          version: latest
+          version: v1.54.2
```

Comment:
Why `1.54.2` instead of `1.57.1` which is the [latest version](https://golangci-lint.run/product/changelog/). Per [their docs](https://golangci-lint.run/welcome/install/#github-actions), pinning the version is a good idea for reproducibility.

Reply:
The latest version as of submitting this PR (two weeks ago) was 1.56.2 which was known to be broken.
I am happy to test 1.57.1 when back at my dev machine in a few days.

Reply:
Looks good, reverted to "latest". So this PR currently boils down to increasing the golangci-lint-action to v4. Feel free to merge or close :)